python-package/lightgbm/plotting.py

```diff
@@ -206,8 +206,8 @@ def plot_split_value_histogram(booster, feature, bins=None, ax=None, width_coef=
 
     hist, bins = booster.get_split_value_histogram(feature=feature, bins=bins, xgboost_style=False)
     if np.count_nonzero(hist) == 0:
-        raise ValueError('Cannot plot split value histogram, '
-                         'because feature {} was not used in splitting'.format(feature))
+        raise ValueError("Cannot plot split value histogram, "
+                         f"because feature {feature} was not used in splitting")
```

Comment:
```suggestion
        raise ValueError('Cannot plot split value histogram, '
                         f'because feature {feature} was not used in splitting')
```

Can you please keep single quotes here? I know that the codebase is somewhat inconsistent about this, but that's an issue for another day. For now, I'd prefer to keep the diff for this pull request as small as possible to minimize the impact on [the "git blame" view](https://docs.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file)

Reply:
Sure.